onnx/onnxifi_dummy.c

```diff
@@ -0,0 +1,133 @@
+/*
+ * Dummy implementation of ONNX backend interface for manual test.
+ * Prints the name of the called function and backend name on each call.
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <math.h>
+
+#include "onnxifi.h"
+
+ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI
+onnxGetBackendIDs(onnxBackendID* backendIDs, size_t* numBackends) {
+  if (backendIDs == NULL || numBackends == NULL) {
```

Comment:
as far as the documentation in the onnxifi goes:
if (backendIDs == NULL) => ONNXIFI_STATUS_FALLBACK
if (numBackends == NULL) => ONNXIFI_STATUS_INVALID_POINTER